### DISCUSSION TITLE
Rest API

### DISCUSSION THREAD

**Discussion by gsportelli73** [YetiForceCompany/YetiForceCRM#15127] [How to]
Hi all, i have created a service rest api to insert some entity in crm. 
Now i must insert a product. 
I have seem that category is manged like tree field. 
When i insert product without category i'm success, when i have insert category i have problem.
I don't understand how filed has structured. 
Can you help me ?
Thank you.

**Comment by mariuszkrzaczkowski** [comment]
Send T... value eg T1
Get values from the method ​/webservice​/RestApi​/{moduleName}​/Fields https://doc.yetiforce.com/api/#/BaseModule/a2cf7c7b2b5da1f48caf19abe4069b97
